OneCommunityGlobal/HighestGoodNetworkApp
# Jingyi assign project to user permission management

# Description
Gives the user permission to add any user on the project members page. "Other Links" -> "Projects" -> "Members" -> "Find user input" 

## Related PRS (if any):
This frontend PR is related to the development backend PR.

## Main changes explained:
- Manage the conditional rendering for Member.jsx

## How to test:
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log as admin/Owner user
5. go to "Other Links" -> "Permission Management" -> "Manage User Permissions"
6. In the project management section, grant a volunteer account without the permission "Assign Project To User" permission.
7. log in to this volunteer account and go to "Other Links" -> "Projects" -> "Members" -> "Find user input." Check if this volunteer can assign and unassign a project to a user.

## Note:
Check the Permissions Management spreadsheet for desired behaviors of permissions
https://docs.google.com/spreadsheets/d/1TfiJY9OLDZuyP2UgF0C1wI9tOSLKbS9MiJFH6n1Dsao/edit#gid=0
